drivers/storage/portworx/manifest/manifest.go

```diff
@@ -312,6 +307,35 @@ func fillPrometheusDefaults(
 	}
 }
 
+func fillTelemetryDefaults(
+	rel *Version,
+) {
+	pxVersion, err := version.NewSemver(rel.PortworxVersion)
+	if err != nil || !pxutil.IsCCMGoSupported(pxVersion) {
```

Comment:
Shouldn't this be `err == nil && !pxutil.IsCCMGoSupported(pxVersion)`?
For unreleased builds the PortworxVersion would be `sha_sha`, so the err will be non-nil. We can assume that the unreleased builds are newer versions, while older released versions will have a proper semantic version.